src/utils/database.ts

```diff
@@ -0,0 +1,40 @@
+import { existsSync, readFileSync, writeFileSync, unlinkSync } from 'fs';
+
+export class Database {
```

Comment:
I'm not sure this needs to be a class. We could export each function individually, export an object or export a singleton. What do you think?